pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotControllerHealthCheck.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.controller.api.resources;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import org.apache.commons.lang.StringUtils;
+import com.linkedin.pinot.controller.ControllerConf;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Api(tags = Constants.HEALTH_TAG)
+@Path("/pinot-controller/admin")
+public class PinotControllerHealthCheck {
+
+  @Inject
+  ControllerConf controllerConf;
+
+  @GET
+  @Path("/pinot-controller/admin")
+  @ApiOperation(value = "Check controller health")
+  @ApiResponses(value = {@ApiResponse(code = 200, message = "Good")})
```

Comment:
The current one seems to return null. Not sure why we have "", but I guess it will fail quickly if the health check does not pass.